docs/source/developer_guide/development-workflow.md

```diff
@@ -18,15 +18,58 @@ limitations under the License.
 ## Development Workflow
 This section describes the steps necessary to build Elyra in a development environment. 
 
+#### Requirements
+
+* [Python 3 Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+* [NodeJS 12+](https://nodejs.org/en/)
+* [Yarn](https://yarnpkg.com/lang/en/docs/install)
+
+### Setting up your development environment
+
+* Install Miniconda
+Download and install a [Python 3 version of Miniconda](https://docs.conda.io/en/latest/miniconda.html) according to your Operating System
+
+* Create a new Python environment
+
+    `conda create -n <env-name> python`
+
+    The python version of your environment will match the miniconda version you installed. You can override the default by explicitly setting `python=3.7`, for example.
+
+* Activate the new environment
+
+    `conda activate dev`
```

Comment:
See previous comment + please  use the placeholder (throughout the doc) or an example name (e.g. `dev`)

```
    conda activate <env-name>
```

Reply:
Good catch! I forgot to update that one with the same placeholder as above